src/host/ut_host/ReadWaitTests.cpp

```diff
@@ -1,129 +0,0 @@
-// Copyright (c) Microsoft Corporation.

```

Comment:
What happened? Why can we get rid of this?

Reply:
Despite the name of the file, it only tested `SplitToOem`, which doesn't exist anymore. We could replace it with unit tests for the new `InputBuffer` code I added, but given that this is already covered by our DBCS feature tests and that further large changes to our input stack are required to allow for full Unicode support in both Windows Terminal and conhost (which will break such unit tests), I don't think that's worth the effort.